### PR TITLE
Task/add average review rating functionality to products

### PR DESCRIPTION
**Pull request by jennywin**
@cchanningallen

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -11,6 +11,12 @@ def index
 
   def show
     @reviews = Review.where(product_id: @product.id).order("created_at DESC")
+
+    if @reviews.blank?
+      @average_review = 0
+    else
+      @average_review = @reviews.average(:rating).round(2)
+    end
```
Comment (cchanningallen) [inline app/controllers/products_controller.rb]: could also be

```
@average_review = @reviews.blank? ? 0 : @reviews.average(:rating).round(2)
```

or, if you want to keep the `if...else`:

```
@average_review = if @reviews.blank?
  0
else
  @reviews.average(:rating).round(2)
end
```

File: app/views/products/show.html.erb [hunk 2]
```diff
@@ -43,4 +44,4 @@ $('.star-rating').raty({
 	readOnly: true,
 	score: function() { return $(this).attr('data-score') }
 });
```
Comment (cchanningallen) [inline app/views/products/show.html.erb:46]: Let's talk about abstracting these in-page JS chunks out into their own files

**Comment by cchanningallen** [comment]
Good work! +1 :kissing_closed_eyes:

**Comment by jennywin** [comment]
@cchanningallen Danke!

**Pull request merged**